Automattic/vip
# FORNO-1677: Fix dev env network site domains

## Description

Changes the generated domains for network sites to be more user-friendly

E.g.:
```
[subsite-id].[main-site-slug].vipdev.lndo.site
 to
[subsite-id]-[subsite-slug].[main-site-slug].vipdev.lndo.site
```

## Steps to Test

1. Check out PR.
1. Run `npm run build`
2. Create a new dev env for a multisite using `./dist/bin/vip-dev-env-create.js`
3. Run `./dist/bin/vip-dev-env-start.js`
1. Run `./dist/bin/vip-dev-env-sync-sql.js`
1. Verify that network sites' domains match new format and are accessible from the browser
